cli/src/commands/get/comments.rs

```diff
@@ -198,7 +198,12 @@ fn download_comments(
         Ok(get_comments_progress_bar(
             if let Some(dataset_name) = dataset_name {
                 *client
-                    .get_statistics(dataset_name)
+                    .get_statistics(
+                        dataset_name,
+                        &StatisticsRequestParams {
+                            ..Default::default()
+                        },
```

Comment:
You don't actually need to bother with the additional import for this:
```suggestion
                        &Default::default(),
```